projects/fine_tuning/toolbox/fine_tuning_run_fine_tuning_job/tasks/main.yml

```diff
@@ -362,3 +362,45 @@ elif fine_tuning_run_fine_tuning_job_workload == 'ilab' -%}
          > {{ artifact_extra_logs_dir }}/artifacts/pod_{{ item | replace('pod/', '') }}.log
     failed_when: false
     loop: "{{ job_pod_name_cmd.stdout_lines }}"
+
+  - name: Retrieve Pod files to artifacts directory
+    when: fine_tuning_run_fine_tuning_job_retrieve_files | bool
+    block:
+      - name: Create the Pod artifacts directory
```

Comment:
can you move `- name` to the same level as `block`, for consistency with the rest of the repo (applies til the bottom of this file)